app/controllers/db_entries_controller.rb

```diff
@@ -178,8 +178,12 @@ def og_db_entry_meta
     if @resource.object.respond_to?(:description_ru)
       og description: @resource.description_meta
     end
-    if @resource.anime? && @resource.id <= 51153
-      og image: "http://cdn.anime-recommend.ru/previews/#{@resource.id}.jpg"
+    if (@resource.anime? && @resource.id <= 51153) or (@resource.manga? && @resource.id <= 144556)
+      if @resource.anime
```

Comment:
```suggestion
      if @resource.anime?
```
знак вопроса часть имени метода. 
руби выдаст ошибку `NoMethodError: undefined method `anime'`